AzTS Data Table/Readme.md

```diff
@@ -0,0 +1,273 @@
+|CustomField3|string|This field contains custom data.|This field contains custom data.|
+|CustomField4|string|This field contains custom data.|This field contains custom data.|
+|CustomField5|string|This field contains custom data.|This field contains custom data.|
+___
+
+### **Baseline Controls**
+
+|Field Name|Field Type|Description|Sample Value|
+|--|--|--|--|
+|OrgTenantId|string|The tenant ID.|null|
+|JobId|int|Job ID for the scan (YYYYMMDD).|20230414|
+|ControlId|string|The control ID.|Azure_Storage_DP_Encrypt_In_Transit|
+|Description|string|A basic description of what the control is about.|HTTPS protocol must be used for accessing Storage Account resources|
+|Id|string|The control short ID.|AzureStorage160|
+|DisplayName|string|The control name.|Enable Secure transfer to storage accounts|
+|Category|string|Generic security specification of the control.|Encrypt data in transit|
+|ControlRequirements|string|Pre-requisites of the control.|Data must be encrypted in transit and at rest|
+|AssessmentName|string|Assessment name associated with the control.|00x0x00x-x000-0xxx-x0x0-000x000x00x0|
+|PolicyDefinitionId|string|Definition id of the policy associated with the control.|/providers/Microsoft.Authorization/policyDefinitions/000x0000-x000-0000-xx00-00x00xx000x0|
+|ControlSeverity|string|The severity of the control.|Critical|
+|FeatureName|string|Type of feature this control belongs to.|Storage|
+|ControlSettings|string|Settings specific to the control to be provided for the scan.|{"ApplicableOsTypes": ["Windows"]}|
+|Tags|string|Labels that denote the control to be of a specific type and belongs to a specific domain.|["Baseline", "SDL", "TCP", "Automated"]|
+|Enabled|bool|Field to indicate whether the control is enabled or disabled.|true|
+|PolicyDefinitionGuid|string|The policy definition guide.|0x0xxx00-x0x0-000x-xx0x-0000x0000000|
+|ResourceType|string|The type of resources this control scans.|"Microsoft.Storage/storageAccounts"|
+|Remediation|string|Steps to remediate control.|For remediation using PowerShell commands, run the command: 'Set-AzStorageAccount -ResourceGroupName <RGName> -Name <StorageAccountName> -EnableHttpsTrafficOnly $true'. Run 'Get-Help Set-AzStorageAccount -full' for more help.|
+|ControlScanSource|string|The control scan source.|Reader|
+|AssessmentProperties|string|The assessment properties.|{"AssessmentNames": ["000000xx-0xxx-0x00-0x00-00x00x0000x0"]}|
+|CustomPolicyProperties|string|The custom policy properties.|{"PolicyDefinitionIds": ["/providers/Microsoft.Authorization/policyDefinitions/000xx000-xx0x-0000-00dd-000000xx000x"]}|
+|Automated|string|Whether the control is manual or automated.|Yes|
+
+___
+
+### **Control Results**
+
+|Field Name|Field Type|Description|Sample Value|
+|--|--|--|--|
+|JobId|int|JobID for scan (YYYYMMDD).|20230414|
+|subscriptionid|string|The subscription ID.|00000xxx-xx00-0x00-x0x0-00xx00000x00|
```

Comment:
NIT: We need to have some level of consistency in text that we're producing for the world.
Stuff like "subscription id" vs "subscription ID". We need to stick to one and have uniformity throughout the document.
I meant the same thing with respect to spacing and indentation in my earlier comment.

Reply:
Done